ext/civicrm_admin_ui/managed/SavedSearch_Queues.mgd.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+use CRM_CivicrmAdminUi_ExtensionUtil as E;
+
+return [
+  [
+    'name' => 'SavedSearch_Queues',
+    'entity' => 'SavedSearch',
+    'cleanup' => 'always',
+    'update' => 'unmodified',
+    'params' => [
+      'version' => 4,
+      'values' => [
+        'name' => 'Queues',
+        'label' => E::ts('Queues'),
+        'api_entity' => 'Queue',
+        'api_params' => [
+          'version' => 4,
+          'select' => [
+            'id',
+            'name',
+            'type:label',
+            'status:label',
+          ],
+          'orderBy' => [],
+          'where' => [],
+          'groupBy' => [],
+          'join' => [],
+          'having' => [],
+        ],
+      ],
+      'match' => [
+        'name',
+      ],
+    ],
+  ],
+  [
+    'name' => 'SavedSearch_Queues_SearchDisplay_Queues_Table_1',
+    'entity' => 'SearchDisplay',
+    'cleanup' => 'always',
+    'update' => 'unmodified',
+    'params' => [
+      'version' => 4,
+      'values' => [
+        'name' => 'Queues_Table_1',
+        'label' => E::ts('Queues Table 1'),
```

Comment:
No need to make translators deal with this.
```suggestion
        'label' => E::ts('Queues'),
```